include/integratorxx/quadratures/muraknowles.hpp

```diff
@@ -132,4 +134,35 @@ struct quadrature_traits<
 
 };
 
+#else
+
+class MuraKnowlesRadialTraits {
+
+  double R_; ///< Radial scaling factor
+
+public:
+
+  MuraKnowlesRadialTraits(double R = 1.0) : R_(R) { }
+
+  template <typename PointType>
+  inline auto radial_transform(PointType x) const noexcept {
+    return -R_ * std::log(1.0 - x*x*x);
+  }
+
+  template <typename PointType>
+  inline auto radial_jacobian(PointType x) const noexcept {
+    const auto x2 = x*x;
+    return R_ * 3.0 * x2 / (1.0 - x2 * x);
+  }
+
+}; 
+
+template <typename PointType, typename WeightType>
+using MuraKnowles = RadialTransformQuadrature<
+  UniformTrapezoid<PointType,WeightType>,
```

Comment:
Per our [2022 paper](https://doi.org/10.1063/5.0121187), using an uniform trapezoidal grid gives horrible accuracy for the Mura-Knowles scheme.

I looked through the [Mura-Knowles paper](https://doi.org/10.1063/1.471749) but it was not obvious which rule is supposed to be used. I hope its author @pjknowles can clarify.

Reply:
According to Peter's email autoreply, he is away from the office until July 13th, so we shouldn't expect a reply until then.

If you already had the Mura-Knowles grid and assumed uniform weights, we can merge this in and open up a separate ticket for the quadrature.

Reply:
OK, since this is the standard we've used in our performance benchmarks for GauXC, I'm hesitant to remove it completely. Its clear from your results that we should at least expose the Gauss-Chebyshev variant though. I would just open an issue to track for the time being.

Reply:
Here is the code we are using in Molpro for the radial grid evaluation on Mura scheme:
```
     alpha=rsc*bragg(naatom)
     ! assumes that r = - alpha * Log[1- x^idftmr]
     ! suggest use idftmr = 3
     write (6,*) 'alpha, npt, idftmr ',alpha,npt,idftmr
     do ir=1,npt
        qval = dble(ir)/dble(n)
        pt(ir) = -alpha*log(1d0-qval**idftmr)
        wt(ir) =  pt(ir)**2 *alpha*dble(idftmr)* &
             & qval**(idftmr-1)/((1d0-qval**idftmr)*dble(n))
     enddo
     write (6,*) 'pt ',pt(:npt)
     write (6,*) 'wt ',wt(:npt)
```
and `n` is `npt+1`, which I think amounts simple trapezoidal quadrature in the `x` of equation (7) in http://dx.doi.org/10.1063/1.471749, and for the Be atom, this is what is generated:
```
 alpha, npt, idftmr    7.0000000000000000                        48                    3
 pt    5.9499271133939913E-005   4.7600833032953915E-004   1.6066578611874361E-003   3.8089732866334377E-003   7.4413311256047504E-003   1.2863600152286464E-002   2.0437970723343253E-002   3.0529977777593031E-002   4.3509725746882791E-002   5.9753327005997357E-002   7.9644569620570760E-002  0.10357683514824530       0.13195529327830460       0.16519940738611444       0.20374579393428979       0.24805148947543415       0.29859769233640943       0.35589406260236178       0.42048368472012887       0.49294882318131050       0.57391763505319449       0.66407204595981006       0.76415705172765447       0.87499178083047879       0.99748274937600856        1.1326398697743505        1.2815959495630309        1.4456306574120197        1.6262002677869758        1.8249749675453268        2.0438861838399740        2.2851873784253796        2.5515332181111825        2.8460842535535971        3.1726476906379575        3.5358703430273275        3.9415088946346315        4.3968179569776069        4.9111235220893743        5.4966995004301378        6.1701626127089160        6.9548035249514397        7.8847270471719106        9.0128012057250508        10.427589422901853        12.295111781246881        14.988083164786680        19.695799083317340
 wt    6.3191408757371128E-013   1.6178925855460011E-010   4.1478214302682156E-009   4.1457509371301411E-008   2.4736257856667049E-007   1.0652626596586621E-006   3.6641284849845252E-006   1.0694442455644551E-005   2.7541538616163128E-005   6.4278015117145217E-005   1.3857027486190222E-004   2.7986264953446042E-004   5.3525184281221503E-004   9.7758247883188328E-004   1.7164520965350302E-003   2.9130257653514745E-003   4.7998471771280651E-003   7.7072235182964691E-003   1.2098296752658244E-002   1.8615653176199027E-002   2.8143351896444287E-002   4.1889696901313278E-002   6.1498123098954999E-002   8.9196494268593482E-002  0.12799934660813353       0.18198381348671958       0.25666916599665018       0.35954374153109814       0.50080416940484207       0.69440463378479145       0.95956585458830923        1.3229773146815327        1.8220646710507786        2.5099284939617785        3.4629679021480624        4.7929337979063913        6.6665154642672082        9.3381941641618038        13.207427859666060        18.922636048645305        27.580468647622130        41.132941288449700        63.288684869795212        101.72526373304323        174.33040888875871        330.69364765060254        753.72105349957747        2659.6767864598341
```
If more is needed to be sure of what the scheme is, please just ask.

Reply:
Thanks @pjknowles, this confirmed that what we had was "canonical". I've added this as reference data in https://github.com/wavefunction91/IntegratorXX/pull/27/commits/8caf841f685dbd3630333a28d34d5bad2a659799

Reply:
Tracking the `GaussChebyshev2` variant in #51, resolving this conversation for now